src/components/testCaseLanding/qdm/TestCaseList.tsx

```diff
@@ -41,13 +41,13 @@ import {
   buildHighlightingForAllGroups,
 } from "../../../util/cqlCoverageBuilder/CqlCoverageBuilder";
 import { uniqWith } from "lodash";
-
+import checkSpecialCharacters from "../common/checkSpecialCharacters";
 export const IMPORT_ERROR =
   "An error occurred while importing your test cases. Please try again, or reach out to the Help Desk.";
-
 export const coverageHeaderRegex =
   /<h2> .* Clause Coverage: (\d*\.\d+|\d*|NaN)%<\/h2>/i;
-
+export const EXPORT_ERROR_CHARACTERS =
```

Comment:
when I read this variable name, I expected a set of characters. could we change the name to mabe special_character_error_message or something similar? also, fullstop or comma between the words "exported" and "some"